src/org/sosy_lab/java_smt/solvers/z3/Z3SolverContext.java

```diff
@@ -162,6 +162,7 @@ public static synchronized Z3SolverContext create(
       Native.setParamValue(cfg, "PROOF", "true");
     }
     Native.globalParamSet("smt.random_seed", String.valueOf(randomSeed));
+    Native.globalParamSet("model.compact", "false");
```

Comment:
@baierd can you give more details on this option? is this option disabling a `partial model`?

Reply:
It deactivates a new default option that compresses function definitions. Deactivating restores the old standard behavior.